README.md

```diff
@@ -37,80 +56,275 @@ If you make changes to the website, repeat the steps to build the image and star
+   [`src/collections/pages`](src/collections/pages) and [`src/collections/posts`](src/collections/posts). Each `de-DE`
+   directory will also need a directory data file called `de-DE.11tydata.js`. The contents should be identical to the
+   default language directory's `en-CA.11tydata.js` file.
+6. Create a localized version of the posts archive page, following the example of [`src/collections/pages/fr-CA/posts.md`](src/collections/pages/fr-CA/posts.md).
+7. Create a localized version of the 404 page, following the example of [`src/collections/pages/fr-CA/404.md`](src/collections/pages/fr-CA/404.md).
+8. To ensure that the 404 page is displayed in the appropriate language, verify that a redirect block has been added to
    the [netlify.toml](netlify.toml) file for each language following the examples. This feature is described in
    Netlify's [redirects documentation](https://docs.netlify.com/routing/redirects/redirect-options/#custom-404-page-handling).
 
-For more information about how Netlify CMS works with internationalized content, see the [internationalization support documentation](https://www.netlifycms.org/docs/beta-features/#i18n-support).
+For more information about how Decap CMS works with internationalized content, see the [internationalization support documentation](https://www.decapcms.org/docs/beta-features/#i18n-support).
+
+## Disabling Internationalization
+
+If you do not need internationalization support for all content on your site, it can be disabled for a specific
+collection or collections. If you don't need internationalization support at all, you should use the `no-i18n`
+branch as the basis of your project (see [Installation](#installation)).
+
+### Disabling Internationalization for a Specific Collection
+
+If your site is translated into more than two languages but you aren't translating all your content, any empty
+collection will not be output by Eleventy. For example, if your site is configured to use `en-CA`, `fr-CA` and
+`de-DE` but you are not translating your posts into `de-DE`, you can leave the `/src/collections/posts/de-DE`
+directory empty and the German posts index page will not be built. However, if you want to disable _all_
+internationalization for a specific collection, you can do so in one of two ways.
+
+#### Quick Method
+
+1. Deleting all directories but the default locale in the `/src/collections/<collection>/` directory. For example, to
+   disable localization for posts, delete the `/src/collections/posts/fr-CA/` directory.
+2. Modify the [`src/admin/config.yml`](src/admin/config.yml) collection block's `i18n` and `folder` properties for the
+   relevant collection. For example, to disable localization for posts, make the following changes in the `posts`
+   block:
+
+   ```diff
+   - i18n: true
+   + i18n: false
+   - folder: "src/collections/posts/"
+   + folder: "src/collections/posts/en-CA/" # Look for posts to edit in the default locale directory.
+   ```
+
+#### Advanced Method
```

Comment:
Resolved in 44ffbd5. The only relevant change for testing is this, @jobara:

https://github.com/greatislander/trivet/blob/44ffbd5281e5a6e7f6035bc48dcac94bf01bc281/README.md?plain=1#L239

My mistake was setting the `lang` variable, which is optional, instead of `locale`, which is not. I also made the permalink computed data entry a little more concise.